contents/docs/product-analytics/web-vitals.mdx

```diff
@@ -45,4 +45,28 @@ To create your own dashboard from a template:
     videoDark="https://res.cloudinary.com/dmukukwp6/video/upload/v1725633861/posthog.com/contents/dark.mp4"
     alt="Popover icon" 
     classes="rounded"
-/>
\ No newline at end of file
+/>
+
+## Events captured
+
+The SDK groups available metrics in order to reduce the cost. That means each `$web_vitals` event carries one or more metric result. By default we wait up to 5 seconds (since the page load or last set of metrics was sent) in order to try and catch all four metrics in one event. See the section on configuring web vitals capture below for how to change this behaviour.
```

Comment:
```suggestion
The SDK attempts to send as many of the available metrics as possible in each `$web_vitals` event. By default, we wait up to 5 seconds since the page load or last `$web_vitals` event to try and catch all four metrics in one event. See the section on [configuring web vitals](#configuring-web-vitals-autocapture) capture to change this behaviour.
```